### ISSUE TITLE
Keras ZeroPad2D layer

### ISSUE THREAD

**Issue by lhfowl** [IBM/CNN-Cert#3]
Hello, 

I'm trying to certify a model I have trained in pytorch, and then converted to keras. I am using run_cnn in pymain.py, but I'm getting the error: 

"ValueError: Invalid Layer Type"

because of the layer: 

"tensorflow.python.keras.layers.convolutional.ZeroPadding2D"

Is this layer not currently supported in your code? Do you have any suggestions on how to modify the code to get past this?

**Comment by AkhilanB** [comment]
Yes, the code does not currently support ZeroPadding2D layers. I suggest modifying either the 'Model' or 'CNNModel' classes in cnn_bounds_full.py or cnn_bounds_full_core.py respectively. One way to to this might be to modify the '\_\_init\_\_' function so that when a padding layer is encountered, it modifies 'self.pads' for the next layer. This way, I think the rest of the code can be used as is.
